src/compile/data/summary.ts

```diff
@@ -1,172 +0,0 @@
-import {AggregateOp} from '../../aggregate';
-import {Channel} from '../../channel';
-import {SOURCE, SUMMARY} from '../../data';
-import {field, FieldDef} from '../../fielddef';
-import {keys, vals, reduce, hash, Dict, StringSet} from '../../util';
-import {VgData} from '../../vega.schema';
-
-import {FacetModel} from './../facet';
-import {LayerModel} from './../layer';
-import {Model} from './../model';
-
-import {DataComponent, SummaryComponent} from './data';
-
-
-export namespace summary {
-  function addDimension(dims: { [field: string]: boolean }, fieldDef: FieldDef) {
-    if (fieldDef.bin) {
-      dims[field(fieldDef, { binSuffix: '_start' })] = true;
-      dims[field(fieldDef, { binSuffix: '_mid' })] = true;
-      dims[field(fieldDef, { binSuffix: '_end' })] = true;
-
-      // const scale = model.scale(channel);
-      // if (scaleType(scale, fieldDef, channel, model.mark()) === ScaleType.ORDINAL) {
-      // also produce bin_range if the binned field use ordinal scale
-      dims[field(fieldDef, { binSuffix: '_range' })] = true;
-      // }
-    } else {
-      dims[field(fieldDef)] = true;
-    }
-    return dims;
-  }
-
-  export function parseUnit(model: Model): SummaryComponent[] {
-    /* string set for dimensions */
-    let dims: StringSet = {};
-
-    /* dictionary mapping field name => dict set of aggregation functions */
-    let meas: Dict<StringSet> = {};
-
-    model.forEach(function(fieldDef: FieldDef, channel: Channel) {
-      if (fieldDef.aggregate) {
-        if (fieldDef.aggregate === AggregateOp.COUNT) {
-          meas['*'] = meas['*'] || {};
-          /* tslint:disable:no-string-literal */
-          meas['*']['count'] = true;
-          /* tslint:enable:no-string-literal */
-        } else {
-          meas[fieldDef.field] = meas[fieldDef.field] || {};
-          meas[fieldDef.field][fieldDef.aggregate] = true;
-        }
-      } else {
-        addDimension(dims, fieldDef);
-      }
-    });
-
-    return [{
-      name: model.dataName(SUMMARY),
-      dimensions: dims,
-      measures: meas
-    }];
-  }
-
-  export function parseFacet(model: FacetModel): SummaryComponent[] {
-    const childDataComponent = model.child().component.data;
-
-    // If child doesn't have its own data source but has a summary data source, merge
-    if (!childDataComponent.source && childDataComponent.summary) {
-      let summaryComponents = childDataComponent.summary.map(function(summaryComponent) {
-        // add facet fields as dimensions
-        summaryComponent.dimensions = model.reduce(addDimension, summaryComponent.dimensions);
-
-        const summaryNameWithoutPrefix = summaryComponent.name.substr(model.child().name('').length);
```

Comment:
In the new `aggregate.ts` file, there is no longer logic for renaming data.  I guess this part is moved somewhere else.  Where does this go though?